thymleafdemo/src/main/java/com/luv2code/springboot/thymleafdemo/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.luv2code.springboot.thymleafdemo.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.luv2code.springboot.thymleafdemo.dao.EmployeeRepository;
+import com.luv2code.springboot.thymleafdemo.entity.Employee;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+
+	private EmployeeRepository employeeRepository;
+	
+	@Autowired
+	public EmployeeServiceImpl(EmployeeRepository theEmployeeRepository) {
+		employeeRepository = theEmployeeRepository;
+	}
+	
+	@Override
+	public List<Employee> findAll() {
+		return employeeRepository.findAllByOrderByLastNameAsc();
+	}
+
+	@Override
+	public Employee findById(int theId) {
+		Optional<Employee> result = employeeRepository.findById(theId);
+		
+		Employee theEmployee = null;
+		
+		if (result.isPresent()) {
```

Comment:
@avantibonde can be refactored

      result.isPresent() ? result.get() : throw new IllegalStateException("Did not find employee id - " + theId);